### DISCUSSION TITLE
How do I create data for a base type of a specified scope?

### DISCUSSION THREAD

**Discussion by hjkl950217** [AutoFixture/AutoFixture#1148] [Q&A]
I want to create the base type data for the specified scope. What should I do?
For example, the following code:
``` csharp
//no object
int a=fixture.create<int>(min:5,max:20);
string b=fixture.create<string>(minLength:5,maxLength:20);


//object 
Feedback_In tt = fixture.Build<Feedback_In>()
                .With(t => t.PlatformName, minLength:5,maxLength:20)
                .Create();


```

I didn't find any APIs. If the library doesn't provide it, I need to write my own extension method. Can you tell me what interface or object to write based on?

## package info
AutoFixture.AutoNSubstitute 4.11
AutoFixture.SeedExtensions 4.11

## env info
.net core 3.0
UT: xunit

Another question: How do you ` just ignore one of the property` when creating object data? Other property are automatically generated by AutoFixture

**Comment by aivascu** [comment]
@hjkl950217 trying to understand what you're after.
You want to be able to specify that for example all resolved `int`s should be in a range or that all subtypes of a type should have a common setup?

In regards to ignoring some properties you can just invoke `.Without(x => x.IgnoredProperty)` in your `.Build<T>()`.
